third_party/2and3/werkzeug/exceptions.pyi

```diff
@@ -1,152 +1,154 @@
-from typing import Any, Dict, NoReturn
+from typing import Any, Dict, NoReturn, Optional
+
+from werkzeug.wrappers import Response
 
 class HTTPException(Exception):
-    code = ...  # type: Any
-    description = ...  # type: Any
-    response = ...  # type: Any
-    def __init__(self, description=None, response=None): ...
+    code = ...  # type: Optional[int]
+    description = ...  # type: Optional[str]
```

Comment:
@JelleZijlstra you didn't exactly mention making `description` `Optional[str]`, but it sounds as if the `description` is being handled axactly the same way as `code`.

Reply:
Yes, I think this is right. It's not handled exactly the same way as `code` because the `HTTPException` constructor can set the attribute, but `Optional[str]` is still right.